tests/test_actions.py

```diff
@@ -76,11 +76,6 @@ def test_releaseadd(self):
         self.assertEqual(self.call_action("releaseadd", {
             "opsys": "FooBarOS",
         }), 1)
-        self.assertEqual(self.call_action("releaseadd", {
```

Comment:
What is the reason for removing the tests?

Reply:
I remember suggesting removing some of them when parsing the choices for some of the command-line options was moved to argparse, since that was causing exceptions to be thrown before the `call_action()` could return. At that point we would be testing argparse, so it made sense to not do that.

Not sure if that still holds, since that was a while ago.